modules/Microsoft.Web/sites/config-appsettings/deploy.bicep

```diff
@@ -48,7 +48,7 @@ var expandedAppSettings = union(appSettingsKeyValuePairs, azureWebJobsValues, ap
 // =========== //
 // Existing resources //
 // =========== //
-resource app 'Microsoft.Web/sites@2020-12-01' existing = {
+resource app 'Microsoft.Web/sites@2022-03-01' existing = {
```

Comment:
Please make sure to align with @krbar as he previously opened a PR, currently in draft, which is related #2501 and also has the related issue assigned

Reply:
Will do, will ask him to close him PR.

Reply:
@eriqua, the PR was closed.